src/components/memory/RecordItem/RecordItem.tsx

```
@@ -269,14 +269,20 @@ const RecordItem = ({ record, shouldBlur = false }: RecordItemProps) => {
   const handleClick = useCallback(() => {
     // 클릭으로 더보기 메뉴 표시
     if (isMyRecord) {
-      openMoreMenu({
+      const menuOptions: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)